pkg/configgen/configgen.go

```diff
@@ -350,3 +366,11 @@ func (g *ConfigGenerator) parsePorts() (string, string) {
 	}
 	return tcp.String(), udp.String()
 }
+
+// NamespacedName returns the general purpose string representation
+func NamespacedName(authenticator *eapolv1.Authenticator) string {
```

Comment:
This feels like there's already some implementation somewhere we could reuse, maybe?